model_compression_toolkit/qat/keras/quantizer/ste_rounding/symmetric_ste.py

```diff
@@ -157,18 +157,16 @@ def convert2inferable(self) -> Union[iq.WeightsPOTInferableQuantizer, iq.Weights
         if self.power_of_two:
             pot_threshold = 2 ** np.ceil(np.log2(self.quantizer_parameters[THRESHOLD_TENSOR]))
             return iq.WeightsPOTInferableQuantizer(num_bits=self.num_bits,
-                                                   threshold=np.reshape(pot_threshold, self.threshold_shape),
-                                                   signed=C.WEIGHTS_SIGNED,
+                                                   threshold=list(pot_threshold.flatten()),
                                                    per_channel=self.per_channel,
-                                                   channel_axis=self.channel_axis)
+                                                   channel_axis=self.channel_axis,
+                                                   input_rank=len(self.threshold_shape))
         else:
             return iq.WeightsSymmetricInferableQuantizer(num_bits=self.num_bits,
-                                                         threshold=np.reshape(
-                                                             self.quantizer_parameters[THRESHOLD_TENSOR],
-                                                             self.threshold_shape),
-                                                         signed=C.WEIGHTS_SIGNED,
+                                                         threshold=list(self.quantizer_parameters[THRESHOLD_TENSOR].numpy().flatten()),
```

Comment:
Because in the POT it's been converted to a numpy array when rounding it to a POT threshold. Here it's a tf variable that is being converted to numpy first.